includes/payments/functions.php

```diff
@@ -520,6 +520,11 @@ function edd_count_payments( $args = array() ) {
 		if ( false !== $is_date ) {
 
 			$date   = new DateTime( $args['end-date'] );
+
+			if ( $args['end-date'] == $args['start-date'] ) {
+				$date->add( new DateInterval( 'P1D' ) );
```

Comment:
`DateInterval` is a `PHP 5.3+` only class. `DateTime` (used above) is `PHP 5.2+` We need a solution that works on HP 5.2+